mappings/net/minecraft/nbt/NbtOps.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_2509 net/minecraft/nbt/NbtOps
+	COMMENT Used to handle Minecraft NBTs within {@link com.mojang.serialization.Dynamic
+	COMMENT dynamics} for DataFixerUpper, allowing generalized serialization logic
+	COMMENT shared across different type of data structures. Use {@link NbtOps#INSTANCE}
+	COMMENT for the ops singleton.
```

Comment:
```suggestion
	COMMENT shared across different types of data structures. Use {@link NbtOps#INSTANCE}.
```

Reply:
Kind of irrelevant to this PR, which is only a repackage